pkg/loadbalancers/l4.go

```diff
@@ -57,17 +57,17 @@ type L4 struct {
 	scope       meta.KeyType
 	namer       namer.L4ResourcesNamer
 	// recorder is used to generate k8s Events.
-	recorder        record.EventRecorder
-	Service         *corev1.Service
-	ServicePort     utils.ServicePort
-	NamespacedName  types.NamespacedName
-	forwardingRules ForwardingRulesProvider
-	healthChecks    healthchecksl4.L4HealthChecks
-	enableDualStack bool
-	network         network.NetworkInfo
-	networkResolver network.Resolver
-
-	svcLogger klog.Logger
+	recorder         record.EventRecorder
+	Service          *corev1.Service
+	ServicePort      utils.ServicePort
+	NamespacedName   types.NamespacedName
+	forwardingRules  ForwardingRulesProvider
+	healthChecks     healthchecksl4.L4HealthChecks
+	enableDualStack  bool
+	network          network.NetworkInfo
+	networkResolver  network.Resolver
+	enableWeightedLB bool
```

Comment:
nit: (it is already load balancer dir)
```suggestion
	enableWeighted bool
```

Reply:
I see that I repeat myself, sorry :D Dont be bothered with things I mentioned last time